src/components/Pages/_Department.scss

```diff
@@ -211,3 +210,9 @@
 .coa-DepartmentPage__related-link {
   text-decoration: none;
 }
+
+.coa-TopServices__container {
+  max-width: 1160px;
+  margin-left: auto;
+  margin-right: auto;
+}
```

Comment:
This is for the Top Services centering style. They are put into a containing div, so the new div needed additional styling.